docs/docs/styled-components.md

```diff
@@ -112,7 +112,31 @@ export default () => (
 )
 ```
 
-### Enabling user stylesheets with a stable class name
+## Creating Global Styles
+
+Normally, styled components are created for a single CSS class that is isolated from other components. In some cases, it’s helpful (and cleaner!) to create a global style so more than one component can have access to the same styles. In a way, you can create a base stylesheet.
```

Comment:
I felt the words Normally & `it’s helpful (and cleaner!)` a bit less welcoming to my taste. Your words were not wrong, though! Probably best to wait for the @gatsbyjs/learning to give some extra pointers.

This is my take on it.
```suggestion
Styled-components is primarily used for a single CSS class that is isolated from other components. In some cases, you want to override global styling — for example, the default margins of your `body` element.  Styled-components has your back. You can use the `createGlobalStyle` to accomplish this. We advise using `createGlobalStyle` in Layout components, which are shared over multiple pages rather than using it on a single page.
```

Reply:
Absolutely! Great catch. Thanks for the updates. If there's anything needed on my end, please let me know.